uktrade/data-hub-frontend
# Update the dashboard feed to use new API endpoint from Data Hub help centre

## Description of change
We need to stop using the Zen desk API feed for the homepage dashboard and use the new feed from the Data Hub Help centre. 

## Test instructions

You should be able to see the new feed pulling through to the homepage dashboard, it should reflect the announcements you can see on this page - https://datahub-helpcentre.london.cloudapps.digital/updates/announcements/

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
